src/test/java/org/elasticsearch/action/count/CountRequestBuilderTests.java

```diff
@@ -78,7 +78,7 @@ public void testStringQueryToString() {
         CountRequestBuilder countRequestBuilder = client.prepareCount();
         String query = "{ \"match_all\" : {} }";
         countRequestBuilder.setQuery(new BytesArray(query));
-        assertThat(countRequestBuilder.toString(), containsString("\"query\":{ \"match_all\" : {} }"));
+        assertThat(countRequestBuilder.toString(), containsString("\"query\" : {\n    \"match_all\" : { }\n  }"));
```

Comment:
I think it is because QuerySourceBuilder.toString() uses pretty printing. Good point about windows. I'm wondering if I should fix the test to use the system line separator or just remove the test. Any opinion?